### PR TITLE
Format data + Filter By Region

### PR DESCRIPTION
**Pull request by vsmart**
Will output data like this: 

```
  { region: [ 'us-east-2' ],
    blendedCost: '123',
    greenRegion: false },
  { region: [ 'us-gov-west-1' ],
    blendedCost: '123',
    greenRegion: true },
  { region: [ 'us-west-1' ],
    blendedCost: '123',
    greenRegion: false },
  { region: [ 'us-west-2' ],
    blendedCost: '123',
    greenRegion: true } ]
```

👋 @mrchrisadams

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -19,24 +19,44 @@ const ceParams = {
     "UsageQuantity"
   ],
   GroupBy: [{
-    Key: 'AZ',
+    Key: 'REGION',
```
Comment (vsmart) [inline src/index.js]: @mrchrisadams I couldn't find data on specific AZs, only regional data. If you have more granular data from AWS based on specific AZ happy to switch this back.

Comment (mrchrisadams) [reply src/index.js]: Hmm… I think you're right in this case.

I haven't been able to find more granular data than this about AWS myself either,and I don't think it brings up more useful information either. Let's drop it for now 👍

**Comment by mrchrisadams** [comment]
@vsmart Thanks so much for this!

I'm pretty happy with this, but I'll test it out before we meet tomorrow for lunch, so we can discuss in person, and work out next steps 👍

**Comment by mrchrisadams** [comment]
Awesome, thanks Franka!

@all-contributors please add @vsmart for coding the cost breakdown for green/grey spend

**Comment by mrchrisadams** [comment]
@all-contributors please add @vsmart for coding the cost breakdown for green/grey spend

**Pull request merged**